main.tf

```diff
@@ -0,0 +1,68 @@
+variable "graphql_api_name" {
```

Comment:
what happens if we grow the project with dozen of variables, and a lot more infrastructure and modules, would it be nice to have them defined in main.tf?

Reply:
Nope it wouldn't, if the scope of the project gets bigger and we need more modules and variables to define then we should be adding separate files for each module and variables. If variables grow more then variable file can be broken down on the basis of cohesion of variables. I didn't create a separate one as in the scope of this project I didn't think that we need to.